spot_rl_experiments/experiments/skill_test/test_sem_place.py

```diff
@@ -14,25 +21,60 @@
 
     if in_fre_lab:
         # at FRE
-        place_target = "coffee_table"
+        place_target = "office chair"
     else:
         # at NYC
         place_target = "test_desk"
 
     spotskillmanager = SpotSkillManager(use_mobile_pick=False, use_semantic_place=True)
-
     is_local = False
     if enable_estimation_before_place:
+        height_target = place_target
         place_target = None
         is_local = True
 
     # Start testing
     contnue = True
+    INITIAL_ARM_JOINT_ANGLES = [0, -180, 180, 90, 0, -90]
+    episode_ctr = 0
+    # Get EE Pose Initial in rest position
+    spot_pos, spot_ort = spotskillmanager.spot.get_ee_pos_in_body_frame()
+    # Set Orientation as Zero
+    spot_ort = np.zeros(3)
     while contnue:
+        # Open Gripper
+        spotskillmanager.spot.open_gripper()
+        input("Place an object in Spot's gripper and press Enter to continue...")
+        # Place Object and Close Gripper
         rospy.set_param("is_gripper_blocked", 0)
-        spotskillmanager.place(place_target, is_local=is_local, visualize=True)
-        contnue = map_user_input_to_boolean("Do you want to do it again ? Y/N ")
+        episode_log = {"actions": []}  # mypy: ignore-errors
+        spotskillmanager.spot.close_gripper()
+        input("waiting for user to get ready with camera")
+        if place_target is None:
+            spotskillmanager.place(
+                place_target, height_target, is_local=is_local, visualize=True
```

Comment:
why there are two variables -- place_target and height_target? since based on the code, they are refering to the name of the object?

Reply:
I added two variables for function testing. Basically in waypoint estimation, place_target would be "None", but I wanted to test the calculate_height function , so forced the place_target as none and created height_target to test out the function.